app/src/main/res/drawable/card.xml

```diff
@@ -4,10 +4,10 @@
     <item>
         <shape>
             <padding
-                android:top="8dp"
-                android:right="8dp"
-                android:bottom="5dp"
-                android:left="8dp"/>
+                android:top="0dp"
```

Comment:
I agree. It's something I recently noticed when I put all the widgets together and was hoping would be fixed either in this PR or right after anyway. 👍